db/system_keyspace.hh

```diff
@@ -143,6 +143,17 @@ class system_keyspace : public seastar::peering_sharded_service<system_keyspace>
     //  and this node crashes after adding a new IP but before removing the old one. The
     //  record with older timestamp is removed, the warning is written to the log.
     future<> peers_table_read_fixup();
+
+    struct peers_cache: public enable_lw_shared_from_this<peers_cache> {
+        std::unordered_map<gms::inet_address, locator::host_id> ip_to_id;
+        std::unordered_map<locator::host_id, gms::inet_address> id_to_ip;
```

Comment:
Nit: `ip_to_id` and `id_to_ip` are visually very similar to each other. How about changing those to `ip_to_host_id` and `host_id_to_ip`, to make it easier to discern them?

Reply:
Can even use inet to increase Levenshtein distance.